third_party/2and3/tabulate.pyi

```diff
@@ -0,0 +1,16 @@
+# Stub for tabulate: https://bitbucket.org/astanin/python-tabulate
+from typing import Any, Sequence
+
+
+def tabulate(
+    tabular_data: Sequence[Sequence[Any]],
+    headers: Sequence[str] = ...,
```

Comment:
`headers` can also be a `str` or a `Dict[str, str]`.

Reply:
Done.